dcpy/utils/excel.py

```diff
@@ -117,6 +118,36 @@ def apply_cross_file_modifications(
     base_wb.save(out_path)
 
 
+def csv_into_excel(
```

Comment:
maybe `from_csv`? Then we have a clean non-repeating function module.function+name in `excel.from_csv`